libsweep/src/protocol.cc

```diff
@@ -75,8 +78,10 @@ void write_command(serial::device_s serial, const uint8_t cmd[2], error_s* error
   packet.cmdByte2 = cmd[1];
   packet.cmdParamTerm = '\n';
 
-  serial::error_s serialerror = nullptr;
+  // pause for 2ms, so the device is never bombarded with back to back commands
+  std::this_thread::sleep_for(std::chrono::milliseconds(2));
```

Comment:
The sweep device should be capable of processing a few commands back to back. But the response behavior can be unexpected in certain situations. I encountered occasional errors on devices when sending multiple commands rapidly back to back. We will continue to work on improving the device's capabilities, but for now, waiting a few milliseconds between successive commands results in a more robust SDK.